packages/web-components/src/helpers.tests.ts

```diff
@@ -0,0 +1,31 @@
+import qs from 'qs';
+
+/**
+ * Returns a formatted URL for a given Storybook fixture.
+ *
+ * @param id - the Storybook fixture ID
+ * @param args - Story args
+ * @returns - the local URL for the Storybook fixture iframe
+ */
+export function fixtureURL(id: string = 'debug--blank', args?: Record<string, any>): string {
```

Comment:
When I try to debug the test, the storybook renders with an error:
<img width="585" alt="image" src="https://github.com/microsoft/fluentui/assets/9615899/caedf566-c1e6-47ce-aac0-d7c9593fa7e2">

(The `radio` story is still rendered under the error).

Reply:
Interesting - repro steps?

<img width="713" alt="image" src="https://github.com/microsoft/fluentui/assets/13071055/08f4d7ad-5deb-4110-aaef-bf28f8b77484">

Reply:
`yarn playwright test --headed --debug`

imo the fixtureURL should be `components-radio--radio`

Reply:
Thanks for this, I'll look now

Reply:
@miroslavstastny resolved!